### DISCUSSION TITLE
`make boostrap` with a error

### DISCUSSION THREAD

**Discussion by monic-shixi** [babel/babel#12096] [General] [answered]
https://github.com/babel/babel/blob/main/CONTRIBUTING.md#developing

- my env
  - `node`
      `10.15.3`
  - `yarn`
     `1.22.5`
  - macos

error in `.yarn/releases/yarn-2.1.1.cjs`, msg info is `TypeError: r(...).brotliDecompressSync is not a function`.

https://github.com/babel/babel/pull/12031  

i can't find yarn version up to 2.x.x...

**Comment by JLHwung** [comment, marked as answer]
Please update node to `>= 10.19.0`. Yarn 1 will invoke Yarn 2 bundled in the babel repo.

**Reply by monic-shixi** [reply]
yep, it's work after update node to `10.19.0` version. 
https://github.com/babel/babel/blob/main/CONTRIBUTING.md#developing 
should update doc for node version?

**Reply by JLHwung** [reply]
Yes, I have drafted a PR for that purpose: https://github.com/babel/babel/pull/12098

**Reply by monic-shixi** [reply]
nice, thanks.

**Comment by merceyz** [comment]
In V2.2 we report an error that the node version is too old instead of crashing

**Reply by JLHwung** [reply]
Yeah unfortunately we can not update to Yarn 2.2 because of https://github.com/yarnpkg/berry/issues/1794. We are waiting for Yarn 2.3 then.
